nixos/modules/services/mail/stalwart-mail.nix

```diff
@@ -107,7 +107,8 @@ in
       resolver.public-suffix = lib.mkDefault [
         "file://${pkgs.publicsuffix-list}/share/publicsuffix/public_suffix_list.dat"
       ];
-      spam-filter.resource = lib.mkDefault "file://${cfg.package}/etc/stalwart/spamfilter.toml";
+      spam-filter.resource = lib.mkDefault "file://${cfg.package.spam-filter}/spam-filter.toml";
+      spam-filter.auto-update = lib.mkDefault false;
```

Comment:
not sure whether this is advisable, I don't know how stalwart is downloading the `https://` URLs inside the file itself.

Reply:
https://github.com/stalwartlabs/stalwart/discussions/1770

I've asked a question in the discussion channel.